File: common/src/main/java/com/github/fcopardo/easyrest/common/callbacks/AfterClientTaskFailure.java
package com.github.fcopardo.easyrest.common.callbacks;

/**
 * Implement this interface in the summoner class to perform any fallback operations when the rest call fails.
 * Created by Fco Pardo on 8/24/14.
 */
public interface AfterClientTaskFailure {
    void onClientTaskFailed(int error);
}

File: common/src/main/java/com/github/fcopardo/easyrest/common/callbacks/AfterClientTaskFailure.kt
package com.github.fcopardo.easyrest.common.callbacks

/**
 * Implement this interface in the summoner class to perform any fallback operations when the rest call fails.
 * Created by Fco Pardo on 8/24/14.
 */
interface AfterClientTaskFailure {
    fun onClientTaskFailed(error: Int)
}
